test/auto/injectorSpec.js

```diff
@@ -688,6 +688,23 @@ describe('injector', function() {
           expect(log.join('; ')).
             toBe('myDecoratedService:input,dependency1; myService:decInput; dec+origReturn');
         });
+
+
+        it('should allow for decorators to $inject', function() {
```

Comment:
_$inject_  -->  _$injector_ (?)
<sub>I would probably change it to something like: _should allow decorating $injector_</sub>